src/ConcreteOperations/linear_combination.jl

```diff
@@ -0,0 +1,37 @@
+export linear_combination
+
+"""
+    linear_combination(P1::SimpleSparsePolynomialZonotope, P2::SimpleSparsePolynomialZonotope)
+
+compute the linear combination of simple sparse polynomial zonotopes `P1` and `P2`.
+
+### Input
+
+- `P1` -- simple sparse polynomial zonotope
+- `P2` -- simple sparse polynomial zonotope
+
+### Output
+
+linear combination of `P1` and `P2`.
+
+### Notes
+
+The linear combination of two sets ``P₁`` and ``P₂`` is defined as
+
+```math
+\\{λp₁ + (1-λ)p₂ | p₁ ∈ P₁, p₂ ∈ P₂, λ ∈ [0, 1]\\}
```

Comment:
This is correct but I would rather use the convention in https://github.com/JuliaReach/juliareach-days-3-reachathon/blob/master/Challenge_5/Challenge5_PolynomialZonotopes.pdf , so it's easier to read the implementation. Or you can just add a second sentence saying that by shifting the domain from `[0, 1]` to `[-1, 1]`, one gets `\\{1/2(1+λ)p₁ + 1/2(1-λ)p₂ | p₁ ∈ P₁, p₂ ∈ P₂, λ ∈ [-1, 1]\\}`

Reply:
The docstring is definitely not the place to do the math and derive one formulation from the other. I think the one I used is more common? (At least it's the one I've always encountered when talking about convex sets and linear combination) Anyway, I can change it

Reply:
In this case, for instance, we would keep the initial version that you had written (because it is the most common one) and add a little `# Notes` paragraph with the observation that implementation-wise we use the normalized interval representation.